packages/lesswrong/lib/routes.ts

```diff
@@ -373,155 +373,169 @@ addRoute(
   }
 );
 
-if (taggingNameIsSet.get()) {
+
+const usePluralTagName = !isLWorAF && taggingNameIsSet.get();
```

Comment:
Isn't this a duplicate of `usePluralTagName` in instanceSettings.ts?

Reply:
Yes. I feel like there was some reason why but maybe not